### DISCUSSION TITLE
how to tweak ores  in other dimensions

### DISCUSSION THREAD

**Discussion by PlayerJman001** [EwyBoy/OreTweaker#75] [Help Wanted!] [answered]
Like I just wanted to make ancient debris more common on my server

**Comment by EwyBoy** [comment, marked as answer]
Just add a new entry to Ore Tweaker and set the block to spawn to ancient debris and the filler to `minecraft:stone` for example.
That will spawn ancient debris in the overworld. For more info check out the wiki or @ me on Discord.
https://github.com/EwyBoy/OreTweaker/wiki
